the-collab-lab/tcl-10-smart-shopping-list
# Ly ql adding logo and semantic head tags to index.html and each route page

## Description

- Add a logo, title, canonical link and basic description to index.html
 - Add React Helmet as a dependency
- Add title and meta tags to each app page to reflect the semantic content of each page (List.js, AddItem.js, Details.js)

We wanted the app to look a little more special by changing the title and logo that appears in the tab. We wanted to add title and meta tags to semantically describe each page.


## Related Issue
closes #45 
closes #53 
## Acceptance Criteria

- adds correct logo and title
- adds semantic title and meta tags for AddItem.js, Details.js, and List.js

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
| ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |
| ✓ | :sparkles: Update styling     |

## Updates

### Before
<img width="218" alt="Screen Shot 2020-09-23 at 9 37 22 PM" src="https://user-images.githubusercontent.com/50968795/94090984-10d5b300-fde5-11ea-9474-8107a15e8af5.png">

### After
<img width="234" alt="Screen Shot 2020-09-23 at 9 37 10 PM" src="https://user-images.githubusercontent.com/50968795/94090957-ff8ca680-fde4-11ea-9446-53c4a74a19e7.png">

<img width="175" alt="Screen Shot 2020-09-24 at 10 42 19 PM" src="https://user-images.githubusercontent.com/25488451/94220556-37641e80-feb7-11ea-937c-ee9d80847d5d.png">

<img width="178" alt="Screen Shot 2020-09-24 at 10 42 57 PM" src="https://user-images.githubusercontent.com/25488451/94220594-4c40b200-feb7-11ea-8864-622cfdd66b25.png">

<img width="224" alt="Screen Shot 2020-09-25 at 12 05 53 PM" src="https://user-images.githubusercontent.com/25488451/94289933-7c279e00-ff27-11ea-9129-591b99037819.png">

## Testing Steps / QA Criteria

- Make sure index.html has new title tag reading "CartVoyant" and a link with `rel="canonical"` to the deployed Netlify website
- Make sure `List.js` has title "Your Shopping List"
- Make sure `AddItem.js` has title "Add Item"
- Make sure `Details.js` has title "Item Details"
